builtin/mainmenu/dlg_settings_advanced.lua

```diff
@@ -639,12 +639,23 @@ local function create_change_setting_formspec(dialogdata)
 		-- Flags
 		formspec = table.concat(fields)
 				.. "checkbox[0.5," .. height - 0.6 .. ";cb_defaults;"
+				--[[~ "defaults" is the name of a mapgen noise
+				setting. This is a checkbox when editing a
+				mapgen noise setting in "All Settings". ]]
```

Comment:
I think it helps to provide more technical context information rather than where exactly this string is used.
Example:
```suggestion
				--[[~ "defaults". This string describes the default processing
				options for noise settings in Mainmenu -> "All Settings". ]]
```